spdx/parsers/tagvaluebuilders.py

```diff
@@ -602,6 +602,7 @@ def create_package(self, doc, name):
         name - any string.
         Raise CardinalityError if package already defined.
         """
+        self.reset_package()
```

Comment:
While I am skeptical in general of reusing builder instances (should create a new one for each new instance creation!), this does seem like a reasonable quick fix that does not require larger refactoring.